benchmark/run.js

```diff
@@ -36,7 +36,8 @@ if (format === 'csv') {
 
 (function recursive(i) {
   const filename = benchmarks[i];
-  const child = fork(path.resolve(__dirname, filename), cli.optional.set);
+  const child = fork(path.resolve(__dirname, filename), cli.optional.set,
+                     { execArgv: ['--expose_internals'] });
```

Comment:
`common.js` ensures that benchmarks are always forked, so there is no way to execute a single benchmark without `--expose_internals` being automatically applied.